diff --git a/bigvgan.py b/bigvgan.py
index e34bdcf..8a1dc04 100644
--- a/bigvgan.py
+++ b/bigvgan.py
@@ -5,6 +5,10 @@
 #   LICENSE is in incl_licenses directory.
 
 import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))  # F5-TTS submodule usage
+
 import json
 from pathlib import Path
 from typing import Optional, Union, Dict
@@ -419,10 +423,10 @@ class BigVGAN(
         revision: str,
         cache_dir: str,
         force_download: bool,
-        proxies: Optional[Dict],
-        resume_download: bool,
-        local_files_only: bool,
-        token: Union[str, bool, None],
+        proxies: Optional[Dict] = None,
+        resume_download: bool = False,
+        local_files_only: bool = False,
+        token: Union[str, bool, None] = None,
         map_location: str = "cpu",  # Additional argument
         strict: bool = False,  # Additional argument
         use_cuda_kernel: bool = False,
